packages/mdctl-axon-tools/lib/StudyManifestTools.js

```diff
@@ -655,6 +683,32 @@ class StudyManifestTools {
     return data
   }
 
+  async getWorkflowNotifications(org, notificationNames) {
+    if (!notificationNames || !notificationNames.length) {
+      return []
+    }
+    const orgDetails = await org.driver.list('org')
+    return getProperty(orgDetails, 'data[0].configuration.notifications', []).filter(n => notificationNames.includes(n.name))
+  }
+
+  async getWorkflowTemplates(org, endPoints) {
+    if (!endPoints || !endPoints.length) {
+      return []
+    }
+    const client = org.driver.client
```

Comment:
I do not thing you need to get the template data, why is needed it for?

Reply:
You are right. It is not needed. Removed it.